qiita_pet/util.py

```diff
@@ -188,6 +188,9 @@ def get_network_nodes_edges(graph, full_access, nodes=None, edges=None):
     # n[1] is the object
     for n in graph.nodes():
         if n[0] == 'job':
+            # ignoring internal Jobs
```

Comment:
Does this need a new test case?

Reply:
This if is needed as a consequence of now adding the internal jobs to the database; my guess is that originally we didn't have it cause we didn't store them. In other words, the test for these method would need to change if we didn't have this if (as now the jobs will be returned here) AKA it's being tested here.